packages/yew/src/lib.rs

```diff
@@ -296,6 +298,23 @@ pub mod events {
 pub use crate::app_handle::AppHandle;
 use web_sys::Element;
 
+thread_local! {
+    static PANIC_HOOK_IS_SET: Cell<bool> = Cell::new(false);
+}
+
+/// Sets a custom panic hook. Unless a panic hook is set through this function, Yew will
+/// overwrite any existing panic hook when one of the `start_app*` functions are called.
```

Comment:
The thing about the first docstring line being special again.
```suggestion
/// Set a custom panic hook.
/// Unless a panic hook is set through this function, Yew will
/// overwrite any existing panic hook when one of the `start_app*` functions are called.
```